server/base/src/main/java/org/apache/accumulo/server/rpc/TServerUtils.java

```diff
@@ -527,6 +529,12 @@ public static ServerAddress createSaslThreadPoolServer(HostAndPort address, TPro
       serverUser = UserGroupInformation.getLoginUser();
     } catch (IOException e) {
       transport.close();
```

Comment:
Can this close trigger its own IOException?

Reply:
It's enclosed in a try, and said exception would just be logged.